src/edu/washington/escience/myria/daemon/MyriaDriverLauncher.java

```diff
@@ -212,9 +214,60 @@ private static int getDriverMemory(final Configuration conf) throws InjectionExc
   }
 
   /**
+   * Prints the expected configuration, compared to the actual configuration given.
+   * Used for logging command line arguments.
+   */
+  private static void logStartupMessage(Class<? extends Name<?>>[] classes, Injector inj) {
+    final StringBuilder sb = new StringBuilder("MyriaDriverLauncher configuration:\n");
+    for (int i = 0; i < classes.length; i++) {
```

Comment:
If you're not using the loop index elsewhere, you should be able to replace this with a foreach loop.